wallet-webapps-common/src/main/webapp/vue-app/components/WalletNotificationAlert.vue

```diff
@@ -1,11 +1,15 @@
 <template>
   <v-alert
+    id="WalletNotificationAlert"
     v-model="displayAlert"
     :type="alertType"
-    class="paragraph"
+    :icon="alertType === 'warning' ? 'mdi-alert-circle' : ''"
     dismissible>
     <div v-sanitized-html="alertMessage" class="self-align-center"></div>
-    <a :href="transactionHashLink" target="_blank">{{ this.$t('exoplatform.wallet.message.followTransaction',{0: this.walletUtils.getUrlHostName(this.walletUtils.getTransactionEtherscanlink())}) }}</a>
+    <a
+      v-if="followTransaction"
+      :href="transactionHashLink"
+      target="_blank">{{ this.$t('exoplatform.wallet.message.followTransaction',{0: this.walletUtils.getUrlHostName(this.walletUtils.getTransactionEtherscanlink())}) }}</a>
```

Comment:
1. avoid using `this` in template
2. use computed value to have the link, especially when you have method calls inside
3. Add `title` for tooltip about what the link is about

```suggestion
    <a
      v-if="followTransaction"
      :href="transactionHashLink"
      :title="$t('exoplatform.wallet.message.transactionExplorerLink')"
      target="_blank">
        {{ transactionLinkLabel }}
      </a>
```